File: src/CountyData.java
public class CountyData {
    private String countyName;
    private CovidStatistics statistics;
    private Coordinates countyCoords;

    public CountyData(String countyName, CovidStatistics statistics, Coordinates countyCoords) {
        this.countyName = countyName;
        this.countyCoords = countyCoords;
        this.statistics = statistics;
    }

    public String getCountyName() {
        return countyName;
    }

    public CovidStatistics getStatistics() {
        return statistics;
    }

    public Coordinates getCountyCoords() {
        return countyCoords;
    }

}

File: Covid19 Analysis/Covid19 Analysis/CountyData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Covid19_Analysis
{
    class CountyData
    {
        private String countyName { get; set; }
        private CovidStatistics statistics { get; set; }
        private Coordinates countyCoords { get; set; }

        public CountyData(String countyName, CovidStatistics statistics, Coordinates countyCoords)
        {
            this.countyName = countyName;
            this.countyCoords = countyCoords;
            this.statistics = statistics;
        }

    }
}
